todoist_api_python/models.py

```diff
@@ -33,6 +34,7 @@ def from_dict(cls, obj):
             is_inbox_project=obj.get("is_inbox_project"),
             is_shared=obj["is_shared"],
```

Comment:
And yet here it is `is_shared` and not `shared`??? I don't understand how this is not broken. 😅

Reply:
Maybe it's v2 that does this (in v2 models the flag is called `is_shared`) 🤷‍♂️